Makefile

```
@@ -54,17 +56,22 @@ usbrelay.c libusbrelay.c: gitversion.h
 
 clean:
 	rm -f usbrelay
-	rm -f libusbrelay.so
+	rm -f libusbrelay.so.$(USBLIBVER)
+	rm -f libusbrelay.so*
 	rm -f gitversion.h
 	$(MAKE) -C usbrelay_py clean
 
 
 install: usbrelay libusbrelay.so
 	install -d $(DESTDIR)$(LIBDIR)
-	install -m 0755 libusbrelay.so $(DESTDIR)$(LIBDIR)
+	install -m 0755 libusbrelay.so.$(USBLIBVER) $(DESTDIR)$(LIBDIR)
+	( cd $(DESTDIR)$(LIBDIR); ln -s libusbrelay.so.$(USBLIBVER) libusbrelay.so ; ln -s libusbrelay.so.$(USBLIBVER) libusbrelay.so.$(USBMAJOR) )
```

As you wrote in the email, these symlinks are usually created by `ldconfig`. I think it's OK to do it this way, but maybe, the following would be simpler:

```suggestion
	$(LDCONFIG) -n $(DESTDIR)$(LIBDIR)
```

At the top of the makefile, you would add something like:
```Makefile
LDCONFIG = /sbin/ldconfig
```
This works on Debian. If that works also on Fedora, we can leave it like this. Otherwise, we may add some logic to autodetect `ldconfig` location.

This also works, if the link already exists. `ln -s` fails, but if you want to stay with `ln`, we should change this to `ln -sf`.